references/synonyms.md

```diff
@@ -195,26 +63,17 @@ Batch write synonyms.
 
 #### Body
 
-| key          | Value description           |
-|-------------------|-----------------------|
-| **input**         | the [one-way string](/advanced_guides/synonyms.md#the-one-way-association) that is gonna be associated with the synonyms array |
-| **synonyms**         | array of words to associate together in [a multi-way](/advanced_guides/synonyms.md#the-multi-way-association) |
-
-An object with either multi-way string associations or one-way string association.
+An object with every synonym to every term.
 
 #### Example
 ```bash
  curl \
-  -X POST 'http://localhost:7700/indexes/12345678/synonyms/batch' \
-  --data '[
-    {
-      "input": "magician",
-      "synonyms": ["harry potter", "merlin", "illusionist"]
-    },
-    {
-      "synonyms": ["mickey", "mouse"]
-    }
-  ]'
+  -X POST 'http://localhost:7700/indexes/12345678/settings/synonyms' \
+  --data '{
+    "Potter" : ["Harry", "hp"],
+    "Harry" : ["Potter", "hp"],
+    "Magician": ["Harry Potter", "Merlin"]
```

Comment:
```suggestion
    "harry potter": ["hp"],
    "hp": ["harry potter"],
    "magician": ["harry potter", "merlin"]
```

Reply:
- I add an example to illustrate the "A is synonym of B and B is synonym of A" part
- I put them in lowercase so that the user is not confuse with the case.